pkg/relayer/proxy/jsonrpc.go

```diff
@@ -138,14 +138,8 @@ func (jsrv *jsonRPCServer) serveHTTP(ctx context.Context, request *http.Request)
 	// Get the relayRequest payload's `io.ReadCloser` to add it to the http.Request
```

Comment:
Should this be `NopCloser`?

Reply:
`NopCloser` == turn my reader into a no op closer. It returns an `io.ReadCloser` with a no-op `Close` method